src/hooks/useUniswapCurrencyList.ts

```diff
@@ -185,27 +185,30 @@ const useUniswapCurrencyList = (searchQuery: string) => {
         list.push({
           color: colors.yellowFavorite,
           data: unfilteredFavorites,
+          key: 'unfilteredFavorites',
           title: tokenSectionTypes.favoriteTokenSection,
         });
-        if (curatedAssets.length) {
-          list.push({
-            data: curatedAssets,
-            title: tokenSectionTypes.verifiedTokenSection,
-            useGradientText: IS_TESTING === 'true' ? false : true,
-          });
-        }
+      }
+      const curatedAssets = getCurated();
```

Comment:
nit not a blocker: is there any reason `getCurated` needs to be a callback instead of memo?